### DISCUSSION TITLE
Reactive prop, no event on update?

### DISCUSSION THREAD

**Discussion by ithuis** [livewire/livewire#8114] [1. Help] [answered]
I made a prop reactive using #[Reactive]. which works great! But the "reactive" change doesn't seem to fire an update / updated event. 
Am i missing something or is this by design?

**Comment by danie-ramdhani** [comment, marked as answer]
> Client-side users can update public properties in many different ways, most commonly by modifying an input with wire:model on it. (https://livewire.laravel.com/docs/lifecycle-hooks#update)

because it's not directly modified by `wire:model` you can't use updated/updating hooks.

BUT you can use this workaround like in this discussion: https://github.com/livewire/livewire/discussions/7989#discussioncomment-8551184

**Reply by ithuis** [reply]
that workaround works!
